lib/Dialect/ESI/runtime/python/esi/esiCppAccel.pyi

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
```

Comment:
Add comment about why this is checked in instead of autogenerated during build, and how it was generated (i.e. just steal the comment from upstream).